frontend/src/app/App.tsx

```diff
@@ -49,8 +49,16 @@ export const WHOAMI_QUERY = gql`
 
 const App = () => {
   const appInsights = getAppInsights();
-
   const dispatch = useDispatch();
+
+  // Check if the user is logged in, if not redirect to Okta
+  if (process.env.REACT_APP_OKTA_ENABLED === "true") {
+    const accessToken = localStorage.getItem("access_token");
+    if (!accessToken) {
+      throw new Error("Not authenticated, redirecting to Okta...");
+    }
```

Comment:
Would anyone object if we pushed this up to the `stg` environment to test it out? I just want to make sure this doesn't interact with our production Okta instance in a weird way.

Reply:
(to clarify I don't think anything in our codebase would cause strange interactions but I don't necessarily trust Okta not to do something weird)

Reply:
Just deployed to test, if you clear your local storage now and log out of Okta preview and go to https://test.simplereport.gov/app, it should take you to Okta! Going to deploy to stg as well

Reply:
So it actually seems we can't manually trigger stg deploys anymore!